### PR TITLE
Upgrade react-scripts to 3.4.3 and Cypress to 4.12.1

### PR DESCRIPTION
**Pull request by yurloc**

### CODE REVIEW

File: optaweb-employee-rostering-standalone/pom.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
   <properties>
     <application.port>8180</application.port>
     <frontend.project.name>optaweb-employee-rostering-frontend</frontend.project.name>
-    <version.cypress.docker>3.6.0</version.cypress.docker>
+    <version.cypress.docker>4.12.1</version.cypress.docker>
```
Comment (yurloc) [inline optaweb-employee-rostering-standalone/pom.xml:20]: This should always be in sync with the version that we have in package.json but I don't know how to enforce that.

Comment (Christopher-Chianelli) [reply optaweb-employee-rostering-standalone/pom.xml:20]: One could create a program to verify that (either via a maven plugin OR node script), but I don't think one exists (there is one for maintaining project version between package.json and pom.xml, but not dependency version: https://www.npmjs.com/package/sync-pom-version-to-package)

**Review by Christopher-Chianelli — APPROVED** [review]
LGTM

**Comment by Christopher-Chianelli** [comment]
@yurloc I think Jenkins is misconfigured for your repo; it cannot find it apparently: `https://****@github.com/yurloc/optaweb-employee-rostering/` not found. I am guessing your fork is still `https://github.com/yurloc/optashift-employee-rostering/` (before shift -> web rename). So either we update Jenkins to use the fork the PR is based on (if possible) or we need to setup a redirect/rename your fork.

**Comment by Christopher-Chianelli** [comment]
(Chain failing due to compile error in Uberfire; Linux failing due to not being able to find @yurloc repo. Given the GH action passed and it works on my machine, I'm merging).

**Comment by yurloc** [comment]
The Jenkins script needs to be able to handle PRs from forks with a different name. CC @Ginxo.

**Comment by Ginxo** [comment]
@yurloc both Jenkins and GA jobs are already prepared for it

- Jenkins https://github.com/kiegroup/jenkins-pipeline-shared-libraries/blob/c6023c3efa478f61e4829146c5275524b0ab6beb/vars/githubscm.groovy#L197
- GA https://github.com/kiegroup/github-action-build-chain/blob/3c5910161d35557e701eb274994de5d3dc3b4c27/src/lib/git.js#L279

GA is already solving this case because it's prepared for `https://api.github.com/repos/${owner}/${repo}/forks` pagination 
```
[INFO]  Projects taken from branch "fix-npm-audit":
    yurloc/optashift-employee-rostering
```
but Jenkins is not, issue created https://issues.redhat.com/browse/BXMSPROD-997
thanks @yurloc

**Pull request merged**